build/packaging/linux/bin/kscnd

```diff
@@ -177,7 +177,7 @@ start() {
     fi
 
     if [[ ! -z $SC_SUB_BRIDGE ]] && [[ $SC_SUB_BRIDGE -eq 1 ]]; then
-        OPTIONS="$OPTIONS --subbridge --subbridgeport $SC_SUB_BRIDGE_PORT --chaintxperiod $SC_TX_PERIOD --chaintxlimit $SC_TX_LIMIT "
+        OPTIONS="$OPTIONS --subbridge --subbridgeport $SC_SUB_BRIDGE_PORT --chaintxperiod $SC_ANCHORING_PERIOD --chaintxlimit $SC_TX_LIMIT "
```

Comment:
Could you change `chaintxperiod` to `anchroingperiod`, too? =)

Reply:
Good. How about apply the change in the next PR? The change will affect binary and node-tests not only packaging related files.

Reply:
OK. :)